app/server/lib/OIDCConfig.ts

```diff
@@ -123,34 +133,26 @@ export class OIDCConfig {
     });
   }
 
-  private async _generateAndStoreCodeVerifier(req: express.Request, sessions: Sessions) {
-    const permitStore = this._gristServer.getExternalPermitStore();
-    const sessionId = sessions.getSessionIdFromRequest(req);
-    if (!sessionId) { throw new Error('no session available'); }
+  private async _generateAndStoreCodeVerifier(req: express.Request) {
+    const mreq = req as RequestWithLogin;
+    if (!mreq.session) { throw new Error('no session available'); }
     const codeVerifier = generators.codeVerifier();
-    const permit: Permit = {
-      action: LOGIN_ACTION,
-      sessionId,
+    mreq.session.oidc = {
       codeVerifier,
     };
-    const state = await permitStore.setPermit(permit, WAIT_IN_MINUTES * 60 * 1000);
-    return { codeVerifier, state };
+    console.log('mreq.session = ', mreq.session);
```

Comment:
This line probably intended while debugging, and not to commit? There are a few other console.log calls sprinkled in the code.